docs/api/expect.md

```diff
@@ -0,0 +1,1273 @@
+
+## toBeCloseTo
+
+- **Type:** `(value: number, numDigits?: number) => Awaitable<void>`
+
+  Use `toBeCloseTo` to compare floating-point numbers. The optional `numDigits` argument limits the number of digits to check _after_ the decimal point. For example:
+
+  ```ts
+  import { expect, test } from 'vitest'
+
+  test.fails('decimals are not equal in javascript', () => {
+    expect(0.2 + 0.1).toBe(0.3) // 0.2 + 0.1 is 0.30000000000000004
+  })
+
+  test('decimals are rounded to 5 after the point', () => {
+    // 0.2 + 0.1 is 0.30000 | "000000000004" removed
+    expect(0.2 + 0.1).toBeCloseTo(0.3, 5)
+    // nothing from 0.30000000000000004 is removed
+    expect(0.2 + 0.1).not.toBeCloseTo(0.3, 50)
+  })
+  ```
+
+## toBeDefined
+
+- **Type:** `() => Awaitable<void>`
+
+  `toBeDefined` asserts that the value is not equal to `undefined`. Useful use case would be to check if function _returned_ anything.
+
+  ```ts
+  import { expect, test } from 'vitest'
+
+  const getApples = () => 3
+
+  test('function returned something', () => {
+    expect(getApples()).toBeDefined()
+  })
+  ```
+
+## toBeUndefined
+
+- **Type:** `() => Awaitable<void>`
+
+  Opposite of `toBeDefined`, `toBeUndefined` asserts that the value _is_ equal to `undefined`. Useful use case would be to check if function hasn't _returned_ anything.
+
+  ```ts
+  import { expect, test } from 'vitest'
+
+  function getApplesFromStock(stock) {
+    if (stock === 'Bill')
+      return 13
+  }
+
+  test('mary doesn\'t have a stock', () => {
+    expect(getApplesFromStock('Mary')).toBeUndefined()
+  })
+  ```
+
+## toBeTruthy
+
+- **Type:** `() => Awaitable<void>`
+
+  `toBeTruthy` asserts that the value is true, when converted to boolean. Useful if you don't care for the value, but just want to know it can be converted to `true`.
+
+  For example having this code you don't care for the return value of `stocks.getInfo` - it maybe complex object, a string or anything else. The code will still work.
+
+  ```ts
+  import { Stocks } from './stocks'
+  const stocks = new Stocks()
+  stocks.sync('Bill')
+  if (stocks.getInfo('Bill'))
+    stocks.sell('apples', 'Bill')
+  ```
+
+  So if you want to test that `stocks.getInfo` will be truthy, you could write:
+
+  ```ts
+  import { expect, test } from 'vitest'
+  import { Stocks } from './stocks'
+  const stocks = new Stocks()
+
+  test('if we know Bill stock, sell apples to him', () => {
+    stocks.sync('Bill')
+    expect(stocks.getInfo('Bill')).toBeTruthy()
+  })
+  ```
+
+  Everything in JavaScript is truthy, except `false`, `0`, `''`, `null`, `undefined`, and `NaN`.
+
+## toBeFalsy
+
+- **Type:** `() => Awaitable<void>`
+
+  `toBeFalsy` asserts that the value is false, when converted to boolean. Useful if you don't care for the value, but just want to know it can be converted to `false`.
+
+  For example having this code you don't care for the return value of `stocks.stockFailed` - it may return any falsy value, but the code will still work.
```

Comment:
```suggestion
  For example, having this code you don't care for the return value of `stocks.stockFailed` - it may return any falsy value, but the code will still work.
```